app/controllers/keybase_proofs_controller.rb

```diff
@@ -0,0 +1,45 @@
+class KeybaseProofsController < ApplicationController
```

Comment:
I really like that you've split this out into its own controller rather than fit it into Users or Settings, this is great.

Could you bring the `/.well-known` endpoint into this controller as well? If we support more of those endpoints in the future it makes a lot more sense to group code by functionality than by URL. And if (knock on wood) anyone has to debug Keybase they'll be happy to have it all together.

Reply:
sure :)